src/scraper/scrape.py

```diff
@@ -30,6 +32,9 @@ def check_config(config: Dict):
     assert config["s3_bucket"]["public_key"]
     assert config["s3_bucket"]["secret_key"]
 
+    assert config["mobilitydatabase"]["url"]
+    assert config["mobilitydatabase"]["token"]
```

Comment:
are all configs going to be through mobility database?

Reply:
I have mandated that to be there for feeds.jsons falling back